benchmarks/src/jmh/java/software/amazon/jdbc/benchmarks/PluginBenchmarks.java

```diff
@@ -141,7 +194,7 @@ public ConnectionWrapper initAndReleaseWithAuroraHostListPlugin() throws SQLExce
   @Benchmark
   public Statement executeStatementBaseline() throws SQLException {
     try (ConnectionWrapper wrapper = new ConnectionWrapper(
-        useAuroraHostList(),
+        usePlugins("executionTime"),
```

Comment:
does it make sense to include the executionTime plugin in the baseline? I would've thought the baseline would be the time to execute without any plugins

Reply:
The test is to measure the baseline time for executing a statement with the executionTime plugin. Before we execute an statement with any plugin we need to call "createStatement", that's my logic for adding it to the baseline.